update-truffleruby.rb

```diff
@@ -0,0 +1,34 @@
+require "digest"
+
+raise "Usage: #{$0} VERSION" unless ARGV.size == 1
+
+version = ARGV[0]
+
+def replace_line(file, pattern, replacement)
+  lines = File.readlines(file)
+  i = lines.index { |line| pattern =~ line }
+  lines[i] = replacement
+  File.write(file, lines.join)
+end
+
+def insert_after(file, pattern, new_lines)
+  lines = File.readlines(file)
+  i = lines.rindex { |line| pattern =~ line }
+  lines.insert(i+1, *new_lines)
+  File.write(file, lines.join)
+end
+
+replace_line "config/db", /truffleruby_version=/, "truffleruby_version=#{version}\n"
+replace_line "config/known", /truffleruby\[/, "truffleruby[-#{version}]\n"
+
+insert_after "config/known_strings", /^truffleruby/, ["truffleruby-#{version}\n"]
+
+%w[md5 sha512].each { |algorithm|
+  digests = %w[linux-amd64 macos-amd64].map { |platform|
+    basename = "truffleruby-#{version}-#{platform}.tar.gz"
+    archive = File.expand_path("../ruby-versions/pkg/#{basename}")
```

Comment:
This is script is specific to your own working environment, isn't it? I am looking at this line...

Reply:
Yes, it is. I wasn't too keen on downloading the archives once more when I have them locally already. But I can download them here if you think it's better, or expect the files to be in the current directory, which is more flexible.

Reply:
I think we could download when not in current directory (or directory passed as a param?)...

Reply:
I still keep wondering where we should locate this file... But maybe for now it's OK to keep it where it is...

Reply:
Good idea, I now added the directory as a command-line argument.